### DISCUSSION TITLE
Clover GUI Screen Distorted on Dell Poweredge T620

### DISCUSSION THREAD

**Discussion by dshanks01** [CloverHackyColor/CloverBootloader#633] [Q&A] [answered]
I'm using Clover 5155 to boot Linux from an NVME drive attached to a PCIe card in a Dell Poweredge T620. Booting works fine, but the Clover GUI is distorted beyond usefulness. See the link below for a picture of the distorted screen. Has anyone else seen this problem and know how to resolve it? I assume Clover is trying to put the integrated VGA port into an unsupported mode, but I haven't a clue how to configure Clover to use a supported mode (nor do I know what modes the T620's integrated graphics support - Dell's documentation is short on details like that). Any help is appreciated.

Following suggestions in the Clover documentation, I looked in preboot.log, linked below, to see what resolution Clover is setting. It's 1680x1050, which is the same resolution that Linux uses on the same VGA port. I tried setting the resolution to 1024x768 and to 1280x1024. I get very similar distortion with those settings. I also tried setting TextOnlly to true and still get a distorted screen. It seems that whatever mode I try, Clover cannot correctly configure the video hardware.

Finally, before someone tells me that that the latest BIOS for the T620 allows booting from an NVMe, yes that's true. But only some NVMes, specifically Dell branded NVMes and similar. The BIOS does not recognize my Samsung 970 EVO Plus. 😒

[Distorted GUI](https://shankscloud.dynu.net/index.php/s/nD7oS3KS44oGz8B)
[preboot.log](https://shankscloud.dynu.net/index.php/s/kMPyLMAP6QrYGGd)

**Comment by RHODOK** [comment]
did you ever find a solution to this? im getting exactly the same issue 😓

**Reply by RHODOK** [reply, marked as answer]
Solved it by copying the CSMVideoDXE.efi from off drivers to the UEFI drivers

**Comment by dshanks01** [comment]
Not only did I never find a solution, you're the first person to respond to
the query.

Ultimately, I just live with it. It's not like I boot the server very
often. So I just cross my fingers when the garbled screen appears. :(

**Reply by RHODOK** [reply]
yeah, shame no one said anything :(

**Comment by dshanks01** [comment]
Cool. Thank you for letting me know. I'll have to try that. :)

**Comment by Disbler** [comment]
> I'm using Clover 5155 to boot Linux from an NVME drive attached to a PCIe card in a Dell Poweredge T620. Booting works fine, but the Clover GUI is distorted beyond usefulness. See the link below for a picture of the distorted screen. Has anyone else seen this problem and know how to resolve it? I assume Clover is trying to put the integrated VGA port into an unsupported mode, but I haven't a clue how to configure Clover to use a supported mode (nor do I know what modes the T620's integrated graphics support - Dell's documentation is short on details like that). Any help is appreciated.
> 
> Following suggestions in the Clover documentation, I looked in preboot.log, linked below, to see what resolution Clover is setting. It's 1680x1050, which is the same resolution that Linux uses on the same VGA port. I tried setting the resolution to 1024x768 and to 1280x1024. I get very similar distortion with those settings. I also tried setting TextOnlly to true and still get a distorted screen. It seems that whatever mode I try, Clover cannot correctly configure the video hardware.
> 
> Finally, before someone tells me that that the latest BIOS for the T620 allows booting from an NVMe, yes that's true. But only some NVMes, specifically Dell branded NVMes and similar. The BIOS does not recognize my Samsung 970 EVO Plus. 😒
> 
> [Distorted GUI](https://shankscloud.dynu.net/index.php/s/nD7oS3KS44oGz8B) [preboot.log](https://shankscloud.dynu.net/index.php/s/kMPyLMAP6QrYGGd)

Good evening forum people.

Sorry for my English.

Something similar happened to me and I posted it in this thread.

https://github.com/CloverHackyColor/CloverBootloader/issues/716

I copied the CSMVideoDXE.efi file to the folder indicated by a user and had no solution.

I greatly appreciate any help.
